.github/workflows/build.yml

```
@@ -42,8 +42,17 @@ jobs:
             - name: Lint and format
               run: pnpm lint && pnpm format
 
-            - name: Run tests
-              run: pnpm test
+            - name: "[Regression Tests] - Authentication"
+              run: pnpm test -- src/app/auth
+
+            - name: "[Regression Tests] - Create Diary"
+              run: pnpm test -- src/app/owner/diary/create
+
+            - name: "[Regression Tests] - Create Pet"
+              run: pnpm test -- src/app/owner/pets/create
+
+            - name: "[Regression Tests] - Integration tests"
+              run: pnpm test -- src/tests/integration/password-validation
```

```suggestion
            - name: "[Regression Tests] - Integration tests"
              run: pnpm test -- src/tests/integration/
```
If we're going to run integration as part of regression tests, I think it would be better to specify the whole integration directory

---

Good point! I made the change